catalog.html

```diff
@@ -56,23 +56,201 @@
+                            <span class="form-checkbox__box"></span>
+                            <span class="form-checkbox__text">Queen</span>
+                            </p>
+                        </label>
+                        <label class="form-checkbox">
+                            <input type="checkbox" class="input-hidden form-checkbox__input" id="name" name="name" required>
+                            <p class="form-checkbox__inner">
+                            <span class="form-checkbox__box"></span>
+                            <span class="form-checkbox__text">Robot</span>
+                            </p>
+                        </label>
+                    </div>
+                </details>
+                <details class="faq__item">
+                    <summary class="faq__question">Eyes</summary>
+                    <div class="faq__answer">
+                        <label class="form-radio form-radio--col">
+                            <input class="input-hidden form-radio__input" type="radio" name="yellow" required>
+                            <span class="form-radio__inner">
+                                <span class="form-radio__box"></span>
+                                <span class="form-radio__text">Yellow</span>
+                            </span>
+                        </label>
+                        <label class="form-radio form-radio--col">
+                            <input class="input-hidden form-radio__input" type="radio" name="green" required>
+                            <span class="form-radio__inner">
+                                <span class="form-radio__box"></span>
+                                <span class="form-radio__text">Green</span>
+                            </span>
+                        </label>
+                        <label class="form-radio form-radio--col">
+                            <input class="input-hidden form-radio__input" type="radio" name="mix" required>
+                            <span class="form-radio__inner">
+                                <span class="form-radio__box"></span>
+                                <span class="form-radio__text">Mix</span>
+                            </span>
+                        </label>
+                        <label class="form-radio form-radio--col">
+                            <input class="input-hidden form-radio__input" type="radio" name="orange" required>
+                            <span class="form-radio__inner">
+                                <span class="form-radio__box"></span>
+                                <span class="form-radio__text">Orange</span>
+                            </span>
+                        </label>
+                    </div>
+                </details>
+                <details class="faq__item">
+                    <summary class="faq__question">Face</summary>
+                        <div class="faq__answer">
```

Comment:
Avoid empty blocks. Add checkboxes. or radio buttons for Face, Skin, Background groups

Reply:
done